tests/test.nf.test.snap

```diff
@@ -36,12 +36,18 @@
             "taxonomy.tre:md5,f76fb2d5aa9b0d637234d48175841e0e",
             "malt_index/table0.db - exists: true",
             "malt_index/table0.idx - exists: true",
+            true,
+            true,
+            true,
+            true,
+            "kmcp/database-kmcp-index.log - contains string: true",
+            "kmcp/database-kmcp.tar.gz - minimum file size: false",
```

Comment:
We want to make sure all our assertions are true.

The `tar.gz` process does seem to result in some variability, I would suggest reducing the file size a bit to account of it - I normally round down to the nearest 10,000 or something (e.g. `3200000`)